src/functions-and-arrays.js

```diff
@@ -47,6 +171,25 @@ const wordsCount = [
   'matter'
 ];
 
+
+function howManyTimes (wordsCount, searchWord)  {
+  count = 0;
+  if(!wordsCount.length){
+    return 0
+  }
+  words.forEach(item => {
+    if (item == searchWord) 
+      count+=1;
+      else
+      count = 0
```

Comment:
Imagine the program is iterating over the words array. It finds an instance of the word you are searching for. Now, the count is 1. The next word is not the word searched for. Now, the count is set back to 0. Is this what you would like to happen?